### ISSUE TITLE
Cropping function crops wrong region

### ISSUE THREAD

**Issue by darcangelomauro** [unitaryfund/aquapointer#59]
The function `DensityCanvas.crop_canvas` crops the density in the wrong region.
The issue is in the following part:

```
x_inds = (
    int((self._origin[0] - center[0] - size[0] / 2) / self._dx),
    int((self._origin[0] - center[0] + size[0] / 2) / self._dx),
)
y_inds = (
    int((self._origin[1] - center[1] - size[1] / 2) / self._dx),
    int((self._origin[1] - center[1] + size[1] / 2) / self._dy),
)
```

`origin` and `center` should be swapped, in the following way:
```
x_inds = (
    int((center[0] - self._origin[0] - size[0] / 2) / self._dx),
    int((center[0] - self._origin[0] + size[0] / 2) / self._dx),
)
y_inds = (
    int((center[1] - self._origin[1] - size[1] / 2) / self._dy),
    int((center[1] - self._origin[1] + size[1] / 2) / self._dy),
)
```

also, in the first entry of`y_inds`, the value `self._dx` should be replaced by `self._dy`

**Comment by darcangelomauro** [comment]
In addition, I think the canvas origin should be updated as well after cropping, with the following line somewhere before `self._fill_derived_attributes()`:
```
self._origin = np.array(center) - np.array(size)/2
```
so that the coordinates displayed in the plotting function are the same as before the cropping

**Comment by darcangelomauro** [comment]
created pull request #60 on the subject

**Comment by Misty-W** [comment]
Fixed in #60
